src/backend/rtlsym.d

```diff
@@ -0,0 +1,179 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1994-1998 by Symantec
+ *              Copyright (c) 2000-2016 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     backendlicense.txt
+ * Source:      $(DMDSRC backend/_rtlsym.d)
+ */
+
+module ddmd.backend.rtlsym;
+
+import ddmd.backend.cc : Symbol;
+
+enum RTLSYM
+{
+    throwd,
+    throwc,
+    throwdwarf,
+    monitor_handler,
```

Comment:
Keep in mind that this is a translation from C++ code to D. The idea in translations is to change as little as practical, in order to make finding problems with the translation much, much easier.

Refactoring (and that includes changing naming conventions) comes *after*. I have a lot of experience translating code from language to language, and have learned this lesson the hard way repeatedly.

With these requested changes, I am losing greppability between the C++ and D sources, which will make dev work harder, not easier. Renaming should happen only after **all** the C++ usages have been converted.

I'm going to revert this back to the way it was originally submitted.